app/Main.hs

```diff
@@ -129,5 +42,25 @@ showCols cols = do
   putStrLn "---------------"
   showCols cols
 
+randomNums :: Int -> IO String
+randomNums x = do
+  g <- newStdGen
+  return $ take x (randomRs ('1', '9') g)
+
+numberWang :: IO ()
+numberWang = do
+  num <- randomNums 1
```

Comment:
Do we want user to choose this `1`? If so take it as a parameter from `randomNums` please.

Reply:
We may let user choose it, that's the digit number for the exercise but left as 1 at the moment since the rules of romaji generation for numbers isn't implemented yet.